extensions/apple-reminders/src/menu-bar.tsx

```diff
@@ -281,6 +288,18 @@ export default function Command() {
           onAction={() => launchCommand({ name: "create-reminder", type: LaunchType.UserInitiated })}
         />
 
+        <MenuBarExtra.Submenu title={`Configure List (${list?.title ?? "All"})`} icon={Icon.List}>
```

Comment:
Some thoughts here: wouldn't it be better to show the currently selected list icon instead of a generic icon? It'd look more scannable.

Reply:
I like the idea! Which icon are you referring to? Are you talking about the `Icon.Circle` that matches the color of the selected list, or is there another icon you have in mind?

Reply:
Yes, exactly, I think you can use either `icon={{ source: Icon.Circle, tintColor: list.color }}` or `icon={Icon.Tray}` depending on the list's type 🙂